lib/drmhelper/tests/fakes.py

```diff
@@ -152,3 +152,17 @@ def getAddonInfo(self, key):
         }
     }
 }
+
+TRANS_PATH_ARGS = [
+    "addon.getSetting('DECRYPTERPATH')",
+    'special://xbmcbinaddons/inputstream.adaptive',
+    'special://home/'
+]
+
+TRANSLATED_PATHS = {
+    'Linux': ['/storage/.kodi/cdm',
+              '/storage/.kodi/addons/inputstream.adaptive'],
+    'Windows': ['c:/users/user/appdata/roaming/Kodi/cdm',
```

Comment:
Want to make this capital C:/ instead?

Interesting the slashes are that way too

Reply:
Yeah win/python is happy with forward slashes that way you don't accidentally escape... `'c:\users\nelly"` is a disaster whereas `'c:\users\helly"` is perfectly fine. Double slash works but forward is nicer I think.

I'll make capital C but Windows love it/hate it doesn't care for case when asking for a path or file. Backwards compatibility with DOS I guess.